### PR TITLE
Add teams to Contributor model

### PR DESCRIPTION
**Pull request by sks444**
Closes https://github.com/coala/community/issues/137

### CODE REVIEW

**Comment by sks444** [comment]
This pr would be valid after https://gitlab.com/coala/landing/merge_requests/46/ is merged.

File: data/models.py [hunk 1]
```diff
@@ -8,6 +10,7 @@ class Contributor(models.Model):
     num_commits = models.IntegerField(default=None, null=True)
     reviews = models.IntegerField(default=None, null=True)
     issues_opened = models.IntegerField(default=None, null=True)
+    teams = JSONField(default=None, null=True)
```
Comment (li-boxuan) [inline data/models.py]: I see that you created another Model called Team in https://gitlab.com/coala/landing/blob/86203d6a9b5b840bc851804858744a4281240286/backend/org/models.py and use `teams = models.ManyToManyField(Team)` in Contributor model. Is there any reason for the use of JSONField here?

Comment (sks444) [reply data/models.py]: >Is there any reason for the use of JSONField here?

Actually, I was using JSONField there; But, yesterday update that to ManyToManyField, and I still have to update it here.

Waiting for that to be merged anyway; Could you please review that one :)

File: data/contrib_data.py [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
 
-from community.git import get_org_name
-from data.models import Contributor
+# from community.git import get_org_name
+from data.models import Contributor, Team
 
 
 def get_contrib_data():
     logger = logging.getLogger(__name__)
-    IMPORT_URL = 'https://webservices.' + get_org_name() + '.io/contrib/'
```
Comment (prnvdixit) [inline data/contrib_data.py]: Maybe keep this commented? Would be a newcomer issue then, to just uncomment this afterward. :smile:

File: data/contrib_data.py [hunk 3]
```diff
@@ -2,13 +2,17 @@
...
+from data.webservices import webservices_url
 
 
 def get_contrib_data():
     logger = logging.getLogger(__name__)
-    IMPORT_URL = 'https://webservices.' + get_org_name() + '.io/contrib/'
+    # IMPORT_URL = 'https://webservices.' + get_org_name() + '.io/contrib/'
+
+    # Using temporary rawgit link until the webservices is down
+    IMPORT_URL = ('https://rawgit.com/' + 'sks444/' + webservices_url() +
```
Comment (jayvdb) [inline data/contrib_data.py]: `IMPORT_URL = community.git.webservices_url('contrib')`

`community.git.webservices_url(api_path)` must return a valid URL, either the real webservices endpoints or your fake one.

all of the mess to create the correct URL must go in `webservices_url`, hidden away from business logic

**Comment by jayvdb** [comment]
ack https://github.com/coala/community/commit/5d83d053da31be6585dcfac18b998485c21950fc

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**